From 304e5a88c3cdfa62acf89fbd51b8146463f6e51d Mon Sep 17 00:00:00 2001
From: "Maciej S. Szmigiero" <mail@maciej.szmigiero.name>
Date: Tue, 7 May 2024 22:51:46 +0200
Subject: [PATCH] hwdb: Add a Logitech MX Master 3S (connected via Bolt
 Receiver)

The mouse technically has settable DPI between 200 and 8000, with a step of
50, but 1000 DPI is the default setting.
And if someone customizes the mouse DPI setting they can make a custom hwdb
rule too.

(cherry picked from commit 7b1768f0a01348c66caba798a1dd7617c33e7bbb)

Related: RHEL-5950
---
 hwdb.d/70-mouse.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index c29c134942..6264268e37 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -540,6 +540,10 @@ mouse:bluetooth:v046dpb019:name:MX Master 2S Mouse:*
  MOUSE_WHEEL_CLICK_COUNT=24
  MOUSE_WHEEL_CLICK_COUNT_HORIZONTAL=14
 
+# Logitech MX Master 3S (via Bolt Receiver)
+mouse:usb:v046dpc548:name:Logitech USB Receiver Mouse:*
+ MOUSE_DPI=1000@142
+
 # Logitech MX Ergo
 mouse:usb:v046dp406f:name:Logitech MX Ergo:*
 mouse:usb:v046dpc52b:name:Logitech Unifying Device. Wireless PID:406f:*
